### ISSUE TITLE
opentpd

### ISSUE THREAD

**Issue by opntr** [HardenedBSD/hardenedBSD#94]

**Comment by xmj** [comment]
Implemented in https://github.com/HardenedBSD/hardenedBSD/tree/hardened/current/master-openntpd/ - can we close it?

**Comment by lattera** [comment]
Not yet. There's still things left to do. Once OpenNTPd is merged into hardened/current/master, though, we can close this.

**Comment by opntr** [comment]
commits from hardened/current/master:

~~~
commit 1c73459600d8bd1e2af23825d7ebf77ac94ccd54
Author: Bernard Spil <brnrd@FreeBSD.org>
Date:   Tue Jul 18 15:10:02 2017 +0200

    HBSD: Update build framework and patch OpenNTPd
    
     - Modifications to openntpd Makefile for 6.2
     - Import server.c patch from FreeBSD ports
    
    Signed-off-by: Bernard Spil <bernard.spil@hardenedbsd.org>

commit 4c11fe9537c3d4c4d80dee0f436b48bd9caf2a11
Author: Bernard Spil <brnrd@FreeBSD.org>
Date:   Tue Jul 18 14:56:27 2017 +0200

    HBSD: Update OpenNTPd to 6.2p1
    
    Signed-off-by: Bernard Spil <bernard.spil@hardenedbsd.org>
~~~

**Comment by opntr** [comment]
@Sp1l please reopen this issue, and remove the done flag if I'm wrongly closed them.
